internal/types/array_test.go

```diff
@@ -116,7 +116,7 @@ func TestArrayMinMax(t *testing.T) {
 		"IntMin": {
 			arr:           intArray,
 			minOrMax:      min,
-			expectedValue: int32(2),
+			expectedValue: int64(2),
```

Comment:
Due to removing number ordering because compat test indicated the number order by different types, expected value is the first smallest one on array if there are multiple same value,.